README.md

```diff
@@ -101,6 +101,11 @@ at `http://<MESOS_MASTER_NODE>:8080/`.
 5. `client` - Used to specify a client node where no services are run but libraries are installed to
 run Accumulo/Hadoop clients.
 
+6. `swarmmanager` - If specified, a Swarm manager will be run on this node and swarm workers will
+be started on all worker nodes (except the swarm manager node). It is recommended that the swarm
```

Comment:
Saying `except the swarm manager node` seems to contradict later sentences.  Could write :

```
Sets up [docker swarm] with the manager on this node and joins all worker nodes to this swarm.  When this is set docker is installed on all worker nodes.  It is recommended that ...
```

Reply:
Fixed now